hardware/EvohomeWeb.cpp

```diff
@@ -149,6 +149,8 @@ bool CEvohomeWeb::StartSession()
 
 bool CEvohomeWeb::StartHardware()
 {
+	if (this->Name.substr(0, 3)=="Fit") // attempt to block old FitBit hardware
```

Comment:
Maybe we could find the root cause of the problem inside the evohomeweb class ?
Maybe use the stable domoticz version, add the fitbit, update to the latest beta ?